### PR TITLE
added test10 file

### PR DESCRIPTION
**Pull request by sammikaperera**
Lahiru could you please review this

### CODE REVIEW

File: test10.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello sri lanka
```
Comment (LahiruSammika95) [inline test10.txt]: make this hello country

**Review by LahiruSammika95 — CHANGES_REQUESTED** [review]
Please make request changes

File: test10.txt (hunk 1 above)
Comment (sammikaperera) [reply test10.txt]: Hi Aiya. I have done it

**Review by LahiruSammika95 — COMMENTED** [review]
Good.

**Pull request merged**